src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
due to [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)
![image](https://user-images.githubusercontent.com/80700247/204357346-b13fb0d9-5d9d-4428-a276-8b44c131afa5.png)

Reply:
I set it to body and ul separately, for this task it enough. But maybe there is more useful way to do it?

Reply:
if there are a lot of tags you need to rewrite, it`ll be better to use reset files or UI libs that also can do it
[https://elad2412.github.io/the-new-css-reset/](https://elad2412.github.io/the-new-css-reset/)
but it looks too much for homework)
you can wright your own reset file with tags and styles you need and use it in future PRs

Reply:
Thank's